webservices/args.py

```diff
@@ -249,8 +249,10 @@ def make_seek_args(field=fields.Int, description=None):
     'candidate_id': fields.List(IStr, description=docs.CANDIDATE_ID),
     'state': fields.List(IStr, description=docs.STATE_GENERIC),
     'party': fields.List(IStr, description=docs.PARTY),
-    'min_first_file_date': fields.Date(description='Selects all committees whose first filing was received by the FEC after this date'),
-    'max_first_file_date': fields.Date(description='Selects all committees whose first filing was received by the FEC before this date'),
+    'min_first_file_date': fields.Date(description='Filter committees by earliest first filing'),
+    'max_first_file_date': fields.Date(description='Filter committees by latest first filing'),
+    'min_last_f1_date': fields.Date(description='Filter committees by earliest most recent Form 1 filing'),
```

Comment:
Is the phrase "earliest most recent"  clear to those familiar with this domain? Is the field name also clear?